randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1249,7 +1275,12 @@ Extra - room_id: [33]
   * L1 Hatch to Pump Control Access/Door to Pump Control Unit
   * Extra - door_idx: (76,)
   > Pickup (Missile Tank)
-      Gravity Suit and Morph Ball and Space Jump and Damage Runs (Beginner)
+      All of the following:
+          Gravity Suit and Morph Ball and Damage Runs (Beginner)
```

Comment:
this needs a electric damage requirement, can just be an arbitrary number for now or could just leave it for when we go through all the damage runs stuff